misc/python/materialize/cli/mkrelease.py

```diff
@@ -274,6 +276,67 @@ def update_versions_list(released_version: Version) -> None:
         raise errors.MzRuntimeError("Couldn't determine where to insert new version")
 
 
+def update_upgrade_tests(released_version: Version) -> None:
+    upgrade_dir = Path("./test/upgrade")
+    version = f"v{released_version}"
+    need_upgrade = [
+        str(p) for p in upgrade_dir.glob("*latest_version*") if "smoke" not in str(p)
+    ]
+    if not need_upgrade:
+        return
+    for path in need_upgrade:
+        spawn.runv(["git", "mv", path, path.replace("latest_version", version)])
+
+    mzcompose = upgrade_dir.joinpath("mzcompose.yml")
+    with mzcompose.open() as fh:
+        contents = fh.readlines()
+
+    workflow_version = str(released_version).replace(".", "_")
+    step = f"""
+      - step: workflow
+        workflow: upgrade-from-{workflow_version}
+"""
+    for i, line in enumerate(contents):
+        if "workflow: upgrade-from-latest" in line:
+            contents.insert(i - 2, step)
+            break
+
+    previous = None
+    tests = None
+    insert_workflow = 0
+    for i, line in enumerate(contents):
+        if "PREVIOUS_VERSION:" in line:
+            previous = line.split(":")[1].strip()
+        if "TESTS:" in line:
+            tests = line.split(":")[1].strip()
+
+        if "upgrade-from-latest:" in line:
+            insert_workflow = i - 1
+            workflow = f"""
+  upgrade-from-{workflow_version}:
+    env:
+      PREVIOUS_VERSION: {previous}
+      TESTS: any_version|{tests}|{version}
```

Comment:
`tests` already uncludes `any_version`, so there is no need to mention it explicitly. Your code generates:

```
TESTS: any_version|any_version|v0.6.1|....
```

Reply:
ah right.